code/modules/food_and_drinks/machinery/smartfridge.dm

```diff
@@ -34,6 +40,10 @@
 			for(var/i in 1 to amount)
 				load(new typekey(src))
 
+/obj/machinery/smartfridge/Moved(atom/old_loc, movement_dir, forced, list/old_locs, momentum_change)
+	. = ..()
+	air_update_turf()
```

Comment:
I fed you bad info on the proc to override, we have a helper for this sorta thing
We need to update both the new turf and the old one, which this doesn't do
```suggestion
/obj/machinery/smartfridge/Move(atom/newloc, direct, glide_size_override, update_dir)
	var/turf/old_loc = loc
	. = ..()
	move_update_air(old_loc)
```

Reply:
I made it update air only on wrench/unwrench. It is working right now with the recent change.